src/test/resources/org/apache/sling/feature/cpconverter/handlers/META-INF/vault/privileges.xml

```diff
@@ -17,5 +17,5 @@
 -->
 <privileges xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0">
    <privilege name="sling:replicate"/>
-   <privilege name="sling:test" abstract="true"><contains name="sling:replicate"/><contains name="jcr:read"/></privilege>
+   <privilege name="sling:test" abstract="true"><contains name="sling.replicate"/><contains name="jcr.read"/></privilege>
```

Comment:
@anchela, I think there is a problem in the repoinit parser. It will not parse "with <aggregate>" if the aggregate matches any of:

```
(["a"-"z"] | ["A"-"Z"])+ ":" (["a"-"z"] | ["A"-"Z"])+
"/" (["a"-"z"] | ["A"-"Z"] | ["0"-"9"] | ["-"] | ["_"] | ["."] | ["@"] | [":"] | ["+"] | ["/"]) *
":repository"
```
or does not match:
```
 (["a"-"z"] | ["A"-"Z"] | ["0"-"9"] | ["-"] | ["_"] | ["."] | ["/"] | [":"] | ["*"]) +
```
That is why I changed the names here to not include a ":" - which obviously is wrong.

Reply:
@karlpauls , hmmmmmmmmmmm.... just had a quick look at the repo-init-parser tests... the one for registering privileges don't come with aggregates containing a namespace prefix. so.... could well be.
https://github.com/apache/sling-org-apache-sling-repoinit-parser/blob/master/src/test/resources/testcases/test-42.txt

Reply:
@karlpauls , altered the test-42.txt (and the corresponding output) to have one aggregated privilege replaced by jcr:read => 

{code}
org.apache.sling.repoinit.parser.RepoInitParsingException: 
Encountered "" at line 4, column 59.
Was expecting one of:
    
	at org.apache.sling.repoinit.parser.impl.RepoInitParserService.parse(RepoInitParserService.java:48)
[...]
Caused by: org.apache.sling.repoinit.parser.impl.ParseException: Encountered "" at line 4, column 59.
Was expecting one of:
    
	at org.apache.sling.repoinit.parser.impl.RepoInitParserImpl.generateParseException(RepoInitParserImpl.java:3843)
{code}

Reply:
I think we can keep it like this here for now and fix the repoinit parser.

Reply:
agreed. i reported SLING-10277 for that

Reply:
apart from that: lgtm.
there are still 3 occurrences of \n in DefaultPackagesEventsEmitter... those also got changed from %n to \n in the light of SLING-10121. so maybe it's worth verifying that as well.